<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="plot_layout">
      <item>
       <widget class="QWebView" name="plot_webview">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="settings_layout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="settings_group">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="3" colspan="2">
          <widget class="QSlider" name="alpha_slider">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="alpha_label">
           <property name="text">
            <string>Alpha</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="alpha_spinbox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="datafile_button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3" colspan="2">
          <widget class="QSlider" name="datasize_slider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>25</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLineEdit" name="datafile_line"/>
         </item>
         <item row="4" column="2">
          <widget class="QSpinBox" name="datasize_spinbox">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QCheckBox" name="datasize_check">
           <property name="text">
            <string>Window</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="datafile_label">
           <property name="text">
            <string>Datafile</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buddy">
            <cstring>datafile_line</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QCheckBox" name="autoupdate_check">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Auto update</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="update_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="new_action"/>
    <addaction name="open_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="about_action"/>
    <addaction name="aboutalges_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="new_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="open_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="aboutalges_action">
   <property name="text">
    <string>About ALGES</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>datafile_line</tabstop>
  <tabstop>datafile_button</tabstop>
  <tabstop>alpha_spinbox</tabstop>
  <tabstop>alpha_slider</tabstop>
  <tabstop>datasize_check</tabstop>
  <tabstop>datasize_spinbox</tabstop>
  <tabstop>datasize_slider</tabstop>
  <tabstop>autoupdate_check</tabstop>
  <tabstop>update_button</tabstop>
  <tabstop>plot_webview</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exit_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datafile_line</sender>
   <signal>returnPressed()</signal>
   <receiver>update_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datasize_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>datasize_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datasize_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>datasize_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datasize_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>datasize_spinbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datasize_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>datasize_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
